app/Http/Requests/Backend/BlogTags/StoreApiBlogTagsRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests\Backend\BlogTags;
+
+use App\Http\Requests\Request;
+
+/**
+ * Class StoreBlogTagsRequest.
+ */
+class StoreApiBlogTagsRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
```

Comment:
its not needed as by default it will be true. Hence authorize method is not needed